docs/Project.toml

```diff
@@ -1,2 +1,5 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
+PowerSimulations = "e690365d-45e2-57bb-ac84-44ba829e73c4"
+PowerSystems = "bcd98974-b02a-5e2f-9ee0-a103f5c450dd"
```

Comment:
Why is PowerSystems here? Isn't `PowerSystems` a dependency of `PowerSimulations`? Also if you have `Pkg.build("PowerSimulations");` in the `.travis.yml` file, why do you need it here in the `docs/Project.toml` file again? Just trying to understand what is going on here.

Reply:
I am also not completely clear about why this happens. Without this dependency the docs code throws a dependency error. I noticed PowerModels had to include InfrastructureModels as well.